### ISSUE TITLE
Can't use Python's iota_client on OSX 10.15.7

### ISSUE THREAD

**Issue by EasonC13** [iotaledger-archive/iota.rs#558]
## Bug description

Can't use Python's iota_client on OSX 10.15.7

## Rust version

Which version of Rust are you running?

- Rust version: I have cargo 1.50.0
- Python Version: 3.7.4 (default, Aug 13 2019, 15:17:50)
    - (Install via Anaconda)

## Version

Which version of the library are you using?

I use the package from [this commit](https://github.com/iotaledger/iota.rs/actions/runs/665254286)

## Hardware specification

What hardware are you using?

- Operating system: OSX
- RAM: 16GB
- Cores: 8 cores

## Steps To reproduce the bug

Explain how the maintainer can reproduce the bug.

1. Download `osx-iota-client-py3.7-wheel` file from [this commit](https://github.com/iotaledger/iota.rs/actions/runs/665254286)
2. unzip and pip install it.
3. open Python with the following code.

```
import iota_client

client = iota_client.Client()
```

## Expected behaviour

I hope it works as the js version. (No Error)

## Actual behaviour (Error)

```
---------------------------------------------------------------------------
PanicException                            Traceback (most recent call last)
<ipython-input-20-5ba5747d7db7> in <module>
      2 
      3 # client will connect to testnet by default
----> 4 client = iota_client.Client()
      5 print(client.get_info())

PanicException: called `Result::unwrap()` on an `Err` value: "PoisonError { inner: .. }"
```

**Comment by casparfritzen** [comment]
I have the same problem on macOS 11.2.3 (20D91)

**Comment by Thoralf-M** [comment]
Please provide an url for now like this 
```python
client = iota_client.Client(nodes_name_password=[["https://api.lb-0.testnet.chrysalis2.com/"]])
```
The default nodes aren't added at the moment, but will be fixed with this PR <https://github.com/iotaledger/iota.rs/pull/550>

**Comment by bingyanglin** [comment]
Solved by PR #550.
